translations/ui-requests/en.json

```diff
@@ -159,5 +155,8 @@
   "requestQueue.requests.notFound": "No requests found",
   "cancellationReason": "Cancellation reason",
   "cancellationAdditionalInformation": "Additional information for patron",
-  "printPickSlips": "Print pick slips for {sp}"
+  "printPickSlips": "Print pick slips for {sp}",
+
+  "common.cancel": "Cancel",
```

Comment:
I see that such translations exists in stripes-components and wonder whether it would be better to get them from there rather than introduce new ones here.

Reply:
@VictorSoroka1, I prefer to have own translation in each module and not rely on translation from stripes-components, because in this case no one will know that this translation is used in another module and can change it in scope of some user story.